### ISSUE TITLE
Unable to hide blue focus ring when searching

### ISSUE THREAD

**Issue by tauling** [Confidenceman02/elm-select#187]
When using the search input field, the default blue focus ring appears around the input, which may not align with our application's design aesthetics.

Expected Behavior:
The input field should not display the default blue focus ring when it receives focus.

Actual Behavior:
The input field currently displays a blue focus ring when it receives focus.

![20240712_16:15:35_Selection_001](https://github.com/user-attachments/assets/02028f96-8621-4c3f-b0c8-7cfa078b2662)

**Comment by Confidenceman02** [comment]
Howdy!

Thanks for creating the issue!

Are you using Tailwind by any chance?

**Comment by Confidenceman02** [comment]
Yep! I have seen this issue pop up with tailwind.

Tailwind adds a box shadow to all inputs which causes the blue ring.

The above PR should fix it, however another more hacky way to get around it is to use the following:

```css
input[data-test-id="selectInput"]:focus {
    box-shadow: none;
}
```
Ill get this sucker merged ASAP.

**Comment by Confidenceman02** [comment]
Released in [10.4.5](https://package.elm-lang.org/packages/Confidenceman02/elm-select/10.4.5/)

If you could let me know if this fixes it that would sweet.

Thanks again!

**Comment by tauling** [comment]
many thanks for your quick help, I appreciate it! I installed the new release (10.4.5) and the issue with the blue focus ring is gone! :+1:

**Comment by Confidenceman02** [comment]
You're very welcome!

Awesome result!
